local-cluster/src/cluster_tests.rs

```diff
@@ -231,7 +231,7 @@ pub fn sleep_n_epochs(
     ticks_per_slot: u64,
     slots_per_epoch: u64,
 ) {
-    let num_ticks_per_second = (1000 / duration_as_ms(&config.target_tick_duration)) as f64;
+    let num_ticks_per_second = config.target_tick_duration.as_secs_f64().recip();
```

Comment:
The previous calculation was correct, but simplified by operating on the `Duration` directly